app/src/main/java/com/arygm/quickfix/ui/elements/QuickFixWorkerOverview.kt

```diff
@@ -0,0 +1,149 @@
+package com.arygm.quickfix.ui.elements
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.LocationOn
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme.colorScheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.arygm.quickfix.R
+import com.arygm.quickfix.model.profile.WorkerProfile
+import com.arygm.quickfix.ui.theme.poppinsTypography
+
+@Composable
+fun QuickFixWorkerOverview(
+    workerProfile: WorkerProfile,
+    modifier: Modifier,
+    sizePP: Dp = 32.dp,
```

Comment:
I suggest that all dp values larger than 16 should be multiplied by a screen scaling factor to ensure responsiveness across different screen sizes.

Reply:
Yeah, but if you see where i've used this composable, I've passer `sizePP` as `36.dp * heightRatio.value`
```kotlin
QuickFixWorkerOverview(
                      workerProfile = workerProfile,
                      modifier =
                          Modifier.padding(
                                  vertical = 16.dp * heightRatio.value,
                                  horizontal = 12.dp * widthRatio.value)
                              .testTag("WorkerOverview"),
                      sizePP = 36.dp * heightRatio.value)
``` 

So I think we're good here 😄

Reply:
okayyy indeed this looks good to me then :)

Reply:
Okay indeed this looks good to me then :)